src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -325,27 +325,29 @@ public SinglePassSamProgram makeInstance(final String outbase, final File input,
     private final Set<MetricAccumulationLevel> accumLevelDefault = CollectionUtil.makeSet(MetricAccumulationLevel.ALL_READS);
 
     @Option(shortName="LEVEL", doc="The level(s) at which to accumulate metrics.")
-    public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = new HashSet<MetricAccumulationLevel>(accumLevelDefault);
+    public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = new HashSet<>(accumLevelDefault);
 
     @Option(shortName = "EXT", doc="Append the given file extension to all metric file names (ex. OUTPUT.insert_size_metrics.EXT). None if null", optional=true)
     public String FILE_EXTENSION = null;
 
-    @Option(doc = "List of metrics programs to apply during the pass through the SAM file.")
-    public List<Program> PROGRAM = CollectionUtil.makeList(Program.CollectAlignmentSummaryMetrics, Program.CollectBaseDistributionByCycle,
+    @Option(doc = "Set of metrics programs to apply during the pass through the SAM file.")
+    public Set<Program> PROGRAM = CollectionUtil.makeSet(Program.CollectAlignmentSummaryMetrics, Program.CollectBaseDistributionByCycle,
```

Comment:
Maybe use a `LinkedHashSet` here so that the insertion order is maintained, making the re-constructed command line deterministic?